geopandas/io/file.py

```diff
@@ -124,8 +120,9 @@ def to_file(df, filename, driver="ESRI Shapefile", schema=None, **kwargs):
     if schema is None:
         schema = infer_schema(df)
     with fiona_env():
+        crs = df.crs.to_string() if hasattr(df.crs, "to_string") else df.crs
```

Comment:
when is the to_string method available / not available ?

Reply:
Should be available with the pyproj pin.

Reply:
http://pyproj4.github.io/pyproj/stable/api/crs.html#pyproj.crs.CRS.to_string#pyproj.crs.CRS.to_string